src/Sarif.Viewer.VisualStudio.UnitTests/CodeAnalysisResultManagerTests.cs

```diff
@@ -155,5 +155,83 @@ public void CodeAnalysisResultManager_GetRebaselinedFileName_IgnoresMismatchedFi
             Tuple<string, string>[] remappedPathPrefixes = target.GetRemappedPathPrefixes();
             remappedPathPrefixes.Should().BeEmpty();
         }
+
+        [Fact]
+        public void CodeAnalysisResultManager_GetRebaselinedFileName_WhenUserDoesNotSelectRebaselinedPath_UsesPathFromLogFile()
```

Comment:
This additional test is unrelated to the bug fix. I should have included it in the previous PR.